### PR TITLE
[primTorch] Implement loss function references

### PR DESCRIPTION
**Pull request by rdspring1**
Add Reference:

- mse_loss
- l1_loss

### CODE REVIEW

File: torch/_refs/nn/functional/__init__.py [hunk 1]
```diff
@@ -297,6 +299,38 @@ def _check_reduction_value(reduction: str):
         raise ValueError("{} is not a valid value for reduction".format(reduction))
 
 
+def _get_string_reduction_arg(
+    size_average: Optional[bool], reduce: Optional[bool]
```
Comment (mruberry) [inline torch/_refs/nn/functional/__init__.py:303]: I would just make these both required kwarg-only arguments for now

Also add a comment describing the helper

File: torch/_refs/nn/functional/__init__.py [hunk 2]
```diff
@@ -29,7 +29,9 @@
     "hardshrink",
     "hardtanh",
     "hinge_embedding_loss",
+    "l1_loss",
     "margin_ranking_loss",
+    "mse_loss",
```
Comment (mruberry) [inline torch/_refs/nn/functional/__init__.py:34]: `mse_loss` would be after `mish`

**Review by mruberry — APPROVED** [review]
Cool! Just a couple small cleanup comments inline, and then this needs to be rebased and you can merge it using pytorchbot!
